### ISSUE TITLE
Add CSL Styles to Libreoffice Documentation

### ISSUE THREAD

**Issue by ThiloteE** [JabRef/user-documentation#509]
- [ ] **Using CSL-Styles and JStyles at the same time in one document.**: 
  >>> I tested this PR. The only way that I found how to destroy its functionality was to select a numeric CSL style, to cite / cite-in-text and then to select the default numeric JStyle and cite / cite-in-text and then to delete some of the priorly cited references. Then pressed "Make/Sync Bibliography". This caused the JStyle reference list at the end of the document to update, but the CSL Style reference list to remain unchanged.
  >>
  >> Hey, thanks for trying out. Yes, the jstyle and csl style citation detection, parsing, bibliography section population and syncing pipelines are completely independent of each other and can't be (or rather isn't recommended to be) used together (one of the reasons being the reference mark format being different - Jstyles don't start with "JABREF_" as you may have noticed with csl). After working with jstyles, if you switch back to a csl style and press refresh, the bibliography for csl would update again, keeping the jstyle section independent in that case (i mean to say behavior is symmetrical the other way around as well).
  > 
  > I think it would be good to add this piece of information to the documentation. It is an edge case and not many people will use both csl and jstyle in one document, but when they do, they will run into this.

**Comment by subhramit** [comment]
> * We also need to add the information on how to make reference marks visible in the user documentation.

Yes, we have it in devdocs but not user docs https://devdocs.jabref.org/code-howtos/openoffice/overview.html#citation-types.
However, here is a thought - does a user need to know about reference marks at all?
